src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -812,15 +820,19 @@ def _keys_allowed():
 
     @EndpointDoc("Display RGW Users",
                  responses={200: RGW_USER_SCHEMA})
-    def list(self, daemon_name=None):
-        # type: (Optional[str]) -> List[str]
-        users = []  # type: List[str]
+    def list(self, daemon_name=None, detailed: bool = False):
+        detailed = str_to_bool(detailed)
```

Comment:
what `detailed` means ? , there should be some description to understand this api change.